src/components/summary-list/template.njk

```diff
@@ -6,20 +6,27 @@
     {% endif -%}
   </a>
 {% endmacro -%}
+
+{# Determine if we need 2 or 3 columns #}
+{% set hasActions = false %}
+{% for row in params.rows %}
+  {% set hasActions = row.actions.items if row.actions.items else hasActions %}
+{% endfor -%}
+
 <dl class="govuk-summary-list {%- if params.classes %} {{ params.classes }}{% endif %}"{% for attribute, value in params.attributes %} {{attribute}}="{{value}}"{% endfor %}>
   {% for row in params.rows %}
-    <div class="govuk-summary-list__row">
+    <div class="govuk-summary-list__row {%- if row.classes %} {{ row.classes }}{% endif %}">
       <dt class="govuk-summary-list__key {%- if row.key.classes %} {{ row.key.classes }}{% endif %}">
         {{ row.key.html | safe if row.key.html else row.key.text }}
       </dt>
       <dd class="govuk-summary-list__value {%- if row.value.classes %} {{ row.value.classes }}{% endif %}">
         {{ row.value.html | indent(8) | trim | safe if row.value.html else row.value.text }}
       </dd>
-      {% if row.actions.items %}
+      {% if row.actions.items.length %}
         <dd class="govuk-summary-list__actions {%- if row.actions.classes %} {{ row.actions.classes }}{% endif %}">
           {% if row.actions.items.length == 1 %}
             {{ _actionLink(row.actions.items[0]) | indent(12) | trim }}
-          {% else %}
+          {% elseif row.actions.items.length > 1 %}
```

Comment:
What else could it be, if `row.actions.items.length` is truthy and it's not `1`?

Reply:
You're right, this got refactored slightly higher up so can go now.

Was part of the first commit: https://github.com/alphagov/govuk-frontend/pull/1259/commits/f0df010ac8fec2047431b3223b8e32a2dc81670a